src/server.ts

```diff
@@ -26,14 +27,15 @@ app.use(express.urlencoded({
 	extended: true
 }));
 
-//app.use(parseUserAgentMiddleware);
-app.use(authenticationMiddleware);
+app.use(parseUserAgentMiddleware);
+// app.use(authenticationMiddleware);
```

Comment:
This can probably be re-added and just replaced with NEX account auth using `getNEXDataByPID`. The PNID parts can just be added back later tbh